aws/rust-runtime/aws-config/Cargo.toml

```diff
@@ -26,7 +26,7 @@ aws-types = { path = "../../sdk/build/aws-sdk/sdk/aws-types" }
 time = { version = "0.3.4", features = ["parsing"] }
 tokio = { version = "1.8.4", features = ["sync"] }
 tracing = { version = "0.1" }
-hyper = { version = "0.14.12", default-features = false }
+hyper = { version = "0.14.20", default-features = false }
```

Comment:
Why was the hyper upgrade necessary?

Reply:
It wasn't, I just saw that the server folks were using `.20` so I figured I'd update our stuff too. I can remove this if you like.

Reply:
Follow the RFC on this: https://awslabs.github.io/smithy-rs/design/rfcs/rfc0021_dependency_versions.html